app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -260,6 +260,10 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
 
         components.core.requestInterceptor.setNavigationController(navHost.navController)
 
+        if (settings().pocketRecommendations) {
+            components.core.pocketUpdater.startPeriodicStoriesRefresh()
+        }
+
```

Comment:
So those refreshs, we only want to do them while the HomeActivity is around?

Reply:
For the refresh a workmanager would be used independently of app lifecycle.
Since it's important for the data to be .. "fresh" maybe there's no need to download/persist new data if it won't be used.